metrics/config_observability.go

```diff
@@ -73,6 +73,12 @@ type ObservabilityConfig struct {
 	// the pods via an HTTP server in the format expected by the pprof visualization tool.
 	EnableProfiling bool
 
+	// DisableQPProfiling indicates whether EnableProfiling flag applies to the QueueProxy processes.
```

Comment:
nit: Renamed to `EnableQPProfiling` to be consistent with other flags?

Reply:
I had that initially. But it means default true, which is non-intuitive.